packages/lmd/elasticsearch/transform/pivot_transform/transform.yml

```
@@ -84,5 +84,5 @@ sync:
     delay: 60s
     field: '@timestamp'
 _meta:
-  fleet_transform_version: 1.0.2
+  fleet_transform_version: 1.2.0
   run_as_kibana_system: false
```

```suggestion
  run_as_kibana_system: false
settings:
  # This is required to prevent the transform from clobbering the Fleet-managed mappings.
  deduce_mappings: false
```

This should fix your error of:

`Error: can't install the package: could not zip-install package; API status code = 500; response body = {"statusCode":500,"error":"Internal Server Error","message":"runtime_exception\n\tCaused by:\n\t\tmapper_parsing_exception: Failed to parse mapping: mapper [source.ip] cannot be changed from type [ip] to [keyword]\n\tRoot causes:\n\t\tmapper_parsing_exception: Failed to parse mapping: mapper [source.ip] cannot be changed from type [ip] to [keyword]"}`

---

@andrewkroh Thanks for helping with this error!